rts/Game/CameraHandler.cpp

```diff
@@ -69,8 +69,8 @@ CCameraHandler* camHandler = nullptr;
 // cameras[ACTIVE] is just used to store which of the others is active
 static CCamera cameras[CCamera::CAMTYPE_COUNT];
 
-static uint8_t camControllerMem[CCameraHandler::CAMERA_MODE_LAST][sizeof(CFreeController)];
-static uint8_t camHandlerMem[sizeof(CCameraHandler)];
+alignas (CFreeController) static uint8_t camControllerMem[CCameraHandler::CAMERA_MODE_LAST][sizeof(CFreeController)];
```

Comment:
Looks decent, but perhaps we could get more idiomatic while we're at it?
```suggestion
alignas (CFreeController) static std::byte camControllerMem[CCameraHandler::CAMERA_MODE_LAST][sizeof(CFreeController)];
```

Reply:
done